tinydir.yaml

```diff
@@ -0,0 +1,32 @@
+package:
```

Comment:
This file looks good, the only concern I have though is it doesn't look to be actively maintained (last commit ~2 years ago). Great there are no current vulnerabilities however.

Id establish if these additional packages resolve the falco build issues - if so then we'll decide whether this is okay for wolfi or needs to be private (due to not being actively maintained)

Reply:
I think it is fine.  It's just a small header library with very simple functionality.  The risk is low for having it.